sonic-slave-bullseye/Dockerfile.j2

```diff
@@ -7,6 +7,14 @@ COPY --from=qemu /usr/bin/qemu-arm-static /usr/bin
 FROM {{ prefix }}multiarch/qemu-user-static:x86_64-aarch64-6.1.0-8 as qemu
 FROM {{ prefix }}multiarch/debian-debootstrap:arm64-bullseye
 COPY --from=qemu /usr/bin/qemu-aarch64-static /usr/bin
+{%- elif CONFIGURED_ARCH == "armhf" and CROSS_BUILD_ENVIRON == "y" %}
+FROM {{ prefix }}multiarch/qemu-user-static:x86_64-arm as qemu
```

Comment:
Can the specific version (6.1.0-8) be used here, as done in the other cases above this line?